ArchiSteamFarm/IPC/Startup.cs

```diff
@@ -97,44 +100,41 @@ internal sealed class Startup {
 		// Add support for default root path redirection (GET / -> GET /index.html), must come before static files
 		app.UseDefaultFiles();
 
-		// Add support for static files (e.g. HTML, CSS and JS from IPC GUI)
-		app.UseStaticFiles(
-			new StaticFileOptions {
-				OnPrepareResponse = static context => {
-					if (context.File is { Exists: true, IsDirectory: false } && !string.IsNullOrEmpty(context.File.Name)) {
-						string extension = Path.GetExtension(context.File.Name);
-
-						CacheControlHeaderValue cacheControl = new();
-
-						switch (extension.ToUpperInvariant()) {
-							case ".CSS" or ".JS":
-								// Add support for SRI-protected static files
-								// SRI requires from us to notify the caller (especially proxy) to avoid modifying the data
-								cacheControl.NoTransform = true;
-
-								goto default;
-							default:
-								// Instruct the caller to always ask us first about every file it requests
-								// Contrary to the name, this doesn't prevent client from caching, but rather informs it that it must verify with us first that his cache is still up-to-date
-								// This is used to handle ASF and user updates to WWW root, we don't want from the client to ever use outdated scripts
-								cacheControl.NoCache = true;
-
-								// All static files are public by definition, we don't have any authorization here
-								cacheControl.Public = true;
-
-								break;
-						}
+#if !NETFRAMEWORK && !NETSTANDARD
+		Dictionary<string, string> pluginsPaths = new();
+
+		if (PluginsCore.ActivePlugins?.Count > 0) {
+			foreach (IWebInterface plugin in PluginsCore.ActivePlugins.OfType<IWebInterface>()) {
+				if (string.IsNullOrEmpty(plugin.PhysicalPath) || string.IsNullOrEmpty(plugin.WebPath)) {
+					continue;
+				}
+
+				string staticFilesDirectory = Path.Combine(Path.GetDirectoryName(plugin.GetType().Assembly.Location)!, plugin.PhysicalPath);
 
-						ResponseHeaders headers = context.Context.Response.GetTypedHeaders();
+				if (Directory.Exists(staticFilesDirectory)) {
+					pluginsPaths.Add(staticFilesDirectory, plugin.WebPath);
```

Comment:
What about duplicate web paths in case of multiple plugins?

Reply:
I think this is not our problem, since we have abandoned the fixed paths for plugins (`/plugins/<pluginName>`) in favor of the ability for plugin developers to choose the path themselves, the only thing worth doing in this case in my opinion is to display warnings.

Reply:
I strongly believe that ASF crashing because "this is not our problem" is a bad idea.

Reply:
> I strongly believe that ASF crashing because "this is not our problem" is a bad idea.

ASF doesn't crash, it's just that only the last plugin's interface will be available, so outputting a warning will suffice in this case.

Reply:
Misc and not easily solvable, we could in theory initialize `HashSet<string>` for duplicate web paths, but it's not easy to determine when 2 web paths are the same, e.g. one might end with a slash and other not, it's the same, but we're not going into this rabbit hole.